tests/unit/amo/pages/TestHome.js

```diff
@@ -499,4 +499,108 @@ describe(__filename, () => {
     expect(root.find('.Home-CuratedThemes')).toHaveLength(1);
     expect(root.find('.Home-FeaturedThemes')).toHaveLength(1);
   });
+
+  it('does not render featured "theme" collections when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_ANDROID });
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: false,
+    });
+
+    const collections = [createFakeCollectionAddonsListResponse()];
+    store.dispatch(loadHomeAddons({ collections, shelves: {} }));
+
+    const _getFeaturedCollectionsMetadata = (i18n) => {
+      return [
+        {
+          footerText: i18n.gettext('some footer text'),
```

Comment:
All this stuff is being repeated in each test, but the only thing that's actually needed is `isTheme`. It seems like a helper of a `fakeFeaturedCollectionsMetadata` could eliminate a lot of this duplication and unnecessary code.

Reply:
Yep, that sounds good to me, thanks!